### PR TITLE
Update syntax for "When I go to the relative URL '$relativeURL'" step

### PR DESCRIPTION
**Pull request by vkepin**

### CODE REVIEW

File: docs/modules/plugins/partials/plugin-web-app-steps.adoc [hunk 1]
```diff
@@ -2,6 +2,24 @@
 :popup-alert: https://www.w3schools.com/js/js_popup.asp
 :close-spec: https://w3c.github.io/webdriver/webdriver-spec.html#close-window[close]
 
+=== Page steps
```
Comment (vkepin) [inline docs/modules/plugins/partials/plugin-web-app-steps.adoc:5]: Update documentation

Comment (valfirst) [inline docs/modules/plugins/partials/plugin-web-app-steps.adoc:5]:
```suggestion
=== Steps opening pages
```

File: docs/modules/plugins/partials/plugin-web-app-steps.adoc [hunk 2]
```diff
@@ -2,6 +2,24 @@
...
+
+==== Go to relative URL
+
+A _relative URL_ - points to a file within a web site (like `about.html` or `/products`)
```
Comment (valfirst) [inline docs/modules/plugins/partials/plugin-web-app-steps.adoc:9]: too technical

PS: javadocs may not be a good example to copy

File: docs/modules/plugins/partials/plugin-web-app-steps.adoc [hunk 3]
```diff
@@ -2,6 +2,24 @@
...
+
+Actions performed at this step:
+
+* Builds the absolute URL by concatenating the base URL and the relative URL;
```
Comment (valfirst) [inline docs/modules/plugins/partials/plugin-web-app-steps.adoc:13]: not clear what source of the base URL is

File: docs/modules/plugins/partials/plugin-web-app-steps.adoc [hunk 4]
```diff
@@ -2,6 +2,24 @@
...
+* Loads the page with the absolute URL.
+
+[source,gherkin]
+----
+When I go to relative URL `$relativeURL`
+----
+
+* `relativeURL` - A string value of the relative URL
+
```
Comment (valfirst) [inline docs/modules/plugins/partials/plugin-web-app-steps.adoc:22]: also please add example demoing step usage

**Review by valfirst — CHANGES_REQUESTED** [review]
Also please fix spotless violations

File: vividus-plugin-web-app/src/main/resources/steps/defaults/composite.steps [hunk 5]
```diff
@@ -284,3 +284,7 @@ When I wait until element located by `<locator>` contains text `<text>'
 Composite: Then element located '$locator' disappears in '$timeout'
 !-- WARNING: The step "Then element located '$locator' disappears in '$timeout'" is deprecated and will be removed in VIVIDUS 0.6.0
 Then element located by `<locator>` disappears in `<timeout>`
+
+Composite: When I go to the relative URL '$relativeURL'
+!-- WARNING: The step "When I go to the relative URL '$relativeURL'" is deprecated and will be removed in VIVIDUS 0.7.0
+When I go to relative URL `<$relativeURL>`
```
Comment (ikalinin1) [inline vividus-plugin-web-app/src/main/resources/steps/defaults/composite.steps:290]:
```suggestion
When I go to relative URL `<relativeURL>`
```

File: docs/modules/plugins/partials/plugin-web-app-steps.adoc (hunk 4 above)
Comment (vkepin) [reply docs/modules/plugins/partials/plugin-web-app-steps.adoc:41]: done

File: docs/modules/plugins/partials/plugin-web-app-steps.adoc [hunk 6]
```diff
@@ -2,6 +2,43 @@
 :popup-alert: https://www.w3schools.com/js/js_popup.asp
 :close-spec: https://w3c.github.io/webdriver/webdriver-spec.html#close-window[close]
 
+=== Steps opening pages
+
+==== Go to relative URL
+
+Navigates the browser to the specific path in the host defined in the _relative URL_ step variable.
```
Comment (uarlouski) [inline docs/modules/plugins/partials/plugin-web-app-steps.adoc:9]: "to the specific path in the host" this statement is not clear

File: docs/modules/plugins/partials/plugin-web-app-steps.adoc [hunk 7]
```diff
@@ -2,6 +2,43 @@
 :popup-alert: https://www.w3schools.com/js/js_popup.asp
 :close-spec: https://w3c.github.io/webdriver/webdriver-spec.html#close-window[close]
 
+=== Steps opening pages
+
+==== Go to relative URL
```
Comment (valfirst) [inline docs/modules/plugins/partials/plugin-web-app-steps.adoc:7]:
```suggestion
==== Open page with relative URL
```

File: docs/modules/plugins/partials/plugin-web-app-steps.adoc (hunk 6 above)
Comment (valfirst) [inline docs/modules/plugins/partials/plugin-web-app-steps.adoc:9]:
```suggestion
Navigates the browser to the specific path in the host defined in the _relative URL_ step parameter.
```

File: vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/PageSteps.java [hunk 8]
```diff
@@ -248,7 +248,7 @@ public void isPageScrolledToAnElement(Locator locator)
      * <p>
      * @param relativeURL A string value of the relative URL
      */
-    @When("I go to the relative URL '$relativeURL'")
+    @When("I go to relative URL `$relativeURL`")
     public void iGoTo(String relativeURL)
```
Comment (uarlouski) [inline vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/PageSteps.java:252]: I think it makes sense to give the method clearer name

File: vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/PageSteps.java [hunk 9]
```diff
@@ -248,7 +248,7 @@ public void isPageScrolledToAnElement(Locator locator)
      * <p>
      * @param relativeURL A string value of the relative URL
      */
```
Comment (uarlouski) [inline vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/PageSteps.java:250]: please don't forget to sync javadoc with main docs

File: docs/modules/plugins/partials/plugin-web-app-steps.adoc [hunk 10]
```diff
@@ -2,6 +2,43 @@
...
+
+Navigates the browser to the specific path in the host defined in the _relative URL_ step variable.
+
+A relative URL can point to a file within a website (like `about.html` or `/products`).
+If the parameter starts with '/' char, navigation will be performed from the root of the host, see examples below.
```
Comment (valfirst) [inline docs/modules/plugins/partials/plugin-web-app-steps.adoc:12]: what about case like `./about.html`?

File: docs/modules/plugins/partials/plugin-web-app-steps.adoc (hunk 4 above)
Comment (valfirst) [reply docs/modules/plugins/partials/plugin-web-app-steps.adoc:41]: still looking for examples

File: vividus-plugin-web-app/src/main/resources/steps/defaults/composite.steps [hunk 11]
```diff
@@ -284,3 +284,7 @@ When I wait until element located by `<locator>` contains text `<text>'
 Composite: Then element located '$locator' disappears in '$timeout'
 !-- WARNING: The step "Then element located '$locator' disappears in '$timeout'" is deprecated and will be removed in VIVIDUS 0.6.0
 Then element located by `<locator>` disappears in `<timeout>`
+
+Composite: When I go to the relative URL '$relativeURL'
```
Comment (valfirst) [inline vividus-plugin-web-app/src/main/resources/steps/defaults/composite.steps:288]: we need to have tests for both new and deprecated steps, so please either add new test or change some existing one to check behaviour of the new step

**Review by valfirst — CHANGES_REQUESTED** [review]

File: vividus-plugin-web-app/src/main/resources/steps/defaults/composite.steps (hunk 11 above)
Comment (vkepin) [reply vividus-plugin-web-app/src/main/resources/steps/defaults/composite.steps:288]: added

File: vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/PageSteps.java (hunk 9 above)
Comment (vkepin) [reply vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/PageSteps.java:265]: updated

File: docs/modules/plugins/partials/plugin-web-app-steps.adoc (hunk 4 above)
Comment (vkepin) [reply docs/modules/plugins/partials/plugin-web-app-steps.adoc:41]: added

File: docs/modules/plugins/partials/plugin-web-app-steps.adoc [hunk 12]
```diff
@@ -2,6 +2,56 @@
...
+|/documents
+|https://mysite.com/documents
+|===
+
+[source,gherkin]
+----
+When I go to relative URL `$relativeURL`
+----
+
+* `relativeURL` - A string value of the relative URL
```
Comment (ikalinin1) [inline docs/modules/plugins/partials/plugin-web-app-steps.adoc:46]:
```suggestion
* `relativeURL` - A string value of the relative URL.
```

**Review by ikalinin1 — APPROVED** [review]

File: vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/PageSteps.java (hunk 8 above)
Comment (vkepin) [reply vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/PageSteps.java]: changed to 'openRelativeUrl'

**Review by valfirst — APPROVED** [review]

**Pull request merged**